lib/dagDisplay.ml

```diff
@@ -920,6 +949,96 @@ and eval_dag_cell_var conf (colspan, align, td) = function
       match td with
       | TDtext s -> VVstring (s : Adef.safe_string :> string)
       | _ -> VVstring "")
+  | [ "index" ] -> (
+      match td with
+      | TDitem (ip, _, _) | TDtext (ip, _) ->
+          (VVstring (string_of_iper ip) : Adef.safe_string :> string)
+      | _ -> VVstring "")
+  | [ "access" ] -> (
+      match td with
+      | TDtext (ip, s) -> VVstring (Util.acces conf base (poi base ip))
+      | _ -> VVstring "")
+  | [ "father"; "access" ] -> (
+      match td with
+      | TDitem (ip, _, _) | TDtext (ip, _) -> (
+          match get_parents (poi base ip) with
+          | Some ifam ->
+              let cpl = foi base ifam in
+              VVstring (Util.acces conf base (poi base (get_father cpl)))
+          | None -> VVstring "")
+      | _ -> VVstring "")
+  | [ "mother"; "access" ] -> (
+      match td with
+      | TDitem (ip, _, _) | TDtext (ip, _) -> (
+          match get_parents (poi base ip) with
+          | Some ifam ->
+              let cpl = foi base ifam in
+              VVstring (Util.acces conf base (poi base (get_mother cpl)))
+          | None -> VVstring "")
+      | _ -> VVstring "")
+  | [ "has_next_sibling" ] -> (
+      match td with
+      | TDitem (ip, _, _) | TDtext (ip, _) -> (
+          match get_parents (poi base ip) with
+          | Some ifam ->
+              let sib = get_children (foi base ifam) in
+              (* array *)
+              let i = ref (-1) in
+              let _ =
+                Array.iteri (fun n s -> if ip = s then i := n else ()) sib
+              in
+              if !i >= 0 && !i < Array.length sib - 1 then VVbool true
```

Comment:
I think you want to do a:
```ocaml
let sib = get_children (foi base ifam) in
VVbool (Array.mem ip sib)
```

Reply:
je ne trouve pas ce code dans lib/dagDisplay.ml!!!

Reply:
Cant locate this code in dagDisplay??

Reply:
humm, I must have reviewed on an older commit and it was moved or was deleted since